packages/features/bookings/Booker/components/EventMeta.tsx

```diff
@@ -44,6 +46,20 @@ export const EventMeta = ({
   const isEmbed = useIsEmbed();
   const hideEventTypeDetails = isEmbed ? embedUiConfig.hideEventTypeDetails : false;
 
+  const { data: eventOwnerDefaultShedule } = trpc.viewer.availability.schedule.get.useQuery({
```

Comment:
Let's fix this typo here as well

```suggestion
  const { data: eventOwnerDefaultSchedule } = trpc.viewer.availability.schedule.get.useQuery({
```